app/src/main/res/layout/administration_ui.xml

```diff
@@ -0,0 +1,296 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+    <RelativeLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <FrameLayout
+            android:id="@+id/pageTopFl"
+            android:layout_width="match_parent"
+            android:layout_height="181dp"
+            android:alpha="0.7"
+            android:background="@drawable/home_header_college_img">
+
+            <TextView
+                android:id="@+id/txtPageTitle"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="45dp"
+                android:layout_marginTop="47sp"
+                android:layout_marginEnd="131dp"
+                android:fontFamily="@font/roboto_bold"
+                android:text="@string/administration"
+                android:textColor="@color/white"
+                android:textSize="23sp"
```

Comment:
define size in dimen.xml

Reply:
yes, bro that's a best practice. Will love, if you work on this issue.